app/controllers/concerns/blacklight/controller.rb

```diff
@@ -133,6 +133,7 @@ def current_or_guest_user
     def discard_flash_if_xhr
       flash.discard if request.xhr?
     end
+    deprecation_deprecate :discard_flash_if_xhr
```

Comment:
It seems like this is going to give people a deprecation message that they can't do anything about.   I'm not sure I understand why we'd do this.

Reply:
Yeah. Since it's completely going away, not being replaced by anything, I wasn't sure what to put. I'll add something. Feel free to help me with the wording on this.